dockerfiles/deb.dockerfile

```diff
@@ -13,70 +16,91 @@
 #   limitations under the License.
 
 ARG BUILD_IMAGE=ubuntu:bionic
```

Comment:
nit: we use version numbers elsewhere (i.e.: 18.04)

Reply:
Ah, yes, I keep mixing up the code names, which is why I used 18.04 in the examples 😅. I can change the examples to use the codenames for consistency